src/actions/appointmentActions.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-unused-vars */
+import axios from 'axios';
```

Comment:
Axios is a huge library. As such we don't wanna be importing it multiple times in our project. I suggest have a single file we you import axios and then just create reusable functions in that file that you can use in your actions

Reply:
Kindly address this